src/pages/home/report/ParticipantLocalTime.js

```diff
@@ -42,14 +42,15 @@ class ParticipantLocalTime extends PureComponent {
 
     getParticipantLocalTime() {
         const reportRecipientTimezone = lodashGet(this.props.participant, 'timezone', CONST.DEFAULT_TIME_ZONE);
-        moment.locale(this.props.preferredLocale);
-        const reportRecipientDay = moment().tz(reportRecipientTimezone.selected).format('dddd');
-        const currentUserDay = moment().tz(this.props.currentUserTimezone.selected).format('dddd');
+        const timezone1 = DateUtils.getLocalMomentFromTimestamp(this.props.preferredLocale, reportRecipientTimezone);
+        const timezone2 = DateUtils.getLocalMomentFromTimestamp(this.props.preferredLocale, this.props.currentUserTimezone.selected);
```

Comment:
We don't use such variable names. Please describe the value it stores.

Reply:
updated the variable names with `reportTimezone` and `currentTimezone` resp.